sdks/python/apache_beam/yaml/examples/transforms/jinja/include/wordCountInclude.yaml

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This examples reads from a public file stored on Google Cloud. This
+# requires authenticating with Google Cloud, or setting the file in
+#`ReadFromText` to a local file.
+#
+# To set up Application Default Credentials,
+# see https://cloud.google.com/docs/authentication/external/set-up-adc.
+#
+# This pipeline reads in a text file, counts distinct words found in the text,
+# then logs a row containing each word and its count.
+
+pipeline:
+  transforms:
+    - name: Read from GCS
+      type: ReadFromText
+      config:
+{% include 'apache_beam/yaml/examples/transforms/jinja/include/submodules/readFromText.yaml' %}
```

Comment:
Rather than just including the config, could we maybe map in whole transforms for simple ones like this?

Since this is an example, we should also explain what we're doing in comments

Reply:
Practically, I don't think it would be common to include just the config like this, especially if it is very simple (and also parameterized)

Reply:
I'd also recommend using more descriptive file names. So for examples:

`readFromTextTransform.yaml` (assuming we put the whole transform there)

or

`mapToFieldsCountConfig.yaml` (assuming we keep that one as config since it is more complicated)

Reply:
1. We don't want to put the whole transform here because it limits us from changing names or inputs.  Wanted to give an example.
2. Ok on names.

Reply:
> We generally don't want to put the whole transform here because it limits us from changing names or inputs.

I don't follow - could you give an example of someething we'd want to change but wouldn't be able to?

> How about a mix? Some transforms, some configs only?

This SGTM - I think the more complicated configs make sense, the one liners seem less useful

Reply:
> I don't follow - could you give an example of someething we'd want to change but wouldn't be able to?
> 
> >

Maybe a mute point now with both options, but if we have the whole transform in the include yaml. We can't change the name or input paths etc unless we pass even more stuff through the jinjia variables.  So the option I went with was to give an example where a user could control or orient the pipeline graph from the main yaml file and then include what configs they needed from a library.  When we push everything to a library, we lose that control, but maybe that is fine.